### PR TITLE
feat: create image and delete icons

### PR DESCRIPTION
**Pull request by luqven**

### CODE REVIEW

**Comment by luqven** [comment]
Current dependencies on/for this PR:
* next
  * **PR #184** 
    * **PR #185** 
      * **PR #193** 
        * **PR #192** 
          * **PR #197** 
            * **PR #194** 
              * **PR #191** 
                * **PR #201** 
                  * **PR #196** 
                    * **PR #198** 
                      * **PR #199**  (this PR)
                        * **PR #200** 
                          * **PR #195** 
      * **PR #187** 
        * **PR #186** 

This comment was auto-generated by [Graphite](https://graphite.dev?utm_source=gh).

File: frontend/src/components/icons/Delete.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import styles from "./Icon.module.scss";
+
+export const DeleteIcon = () => {
+  return (
+    <div className={styles.container}>
+      <svg
+        width="16"
```
Comment (frederickfogerty) [inline frontend/src/components/icons/Delete.tsx:8]: Let's delete the `width` and `height` attributes

**Review by frederickfogerty — CHANGES_REQUESTED** [review]
After removing `width` and `height` this should be good

**Review by frederickfogerty — COMMENTED** [review]
LGTM

**Pull request merged**